tests/test_agentbox.py

```
@@ -789,6 +789,92 @@ def test_a_quoted_false_does_not_turn_codex_loose(self):
+        with tempfile.TemporaryDirectory() as tmp:
+            prof = build_fake_profile(tmp)
+            for web, want in ((True, []), (False, ["main"])):
+                config = json.loads(json.dumps(base))
+                config["web"] = {"enable": web}
+                # No "sessions" key at all: this is the defaulting path.
+                config["users"] = {"agent": {}}
+                spec = mod.Spec(config, prof)
+                rend = mod.Renderer(spec, prof, root=Path(tmp) / f"w{web}")
+                tree = rend.render()
+                seed = [text for path, (text, _) in tree.files.items()
+                        if path.endswith("agent-sessions.json")][0]
```

Ruff (0.16.2)

[warning] Prefer `next(...)` over single element slice

Replace with `next(...)`

(RUF015)